### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by korJiusngPark**

### CODE REVIEW

File: 1/1_2557_pjs.cpp [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+#include "1_InputOutput_1.h"
```
Comment (jukqaz) [inline 1/1_2557_pjs.cpp:2]: @korJiusngPark 
Where is `1_InputOutput_1.h`?

File: 1/2_1000_pjs.cpp [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#include "1_1000.h"
```
Comment (jukqaz) [inline 1/2_1000_pjs.cpp:1]: @korJiusngPark 
Where is `1_1000.h`?

File: 1/3_1001_pjs.cpp [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#include "1.h"
```
Comment (jukqaz) [inline 1/3_1001_pjs.cpp:1]: @korJiusngPark 
Where is `1.h`?

File: 1/4_7287_pjs.cpp [hunk 4]
```diff
@@ -0,0 +1,9 @@
+#include "1.h"
```
Comment (jukqaz) [inline 1/4_7287_pjs.cpp:1]: @korJiusngPark 
Where is `1.h`?

File: 1/5_10172_pjs.cpp [hunk 5]
```diff
@@ -0,0 +1,13 @@
+#include "1.h"
```
Comment (jukqaz) [inline 1/5_10172_pjs.cpp:1]: @korJiusngPark 
Where is `1.h`?

File: 1/6_10718_pjs.cpp [hunk 6]
```diff
@@ -0,0 +1,9 @@
+#include "1.h"
```
Comment (jukqaz) [inline 1/6_10718_pjs.cpp:1]: @korJiusngPark 
Where is `1.h`?

File: 1/7_11718_pjs.cpp [hunk 7]
```diff
@@ -0,0 +1,41 @@
+#include "1.h"
```
Comment (jukqaz) [inline 1/7_11718_pjs.cpp:1]: @korJiusngPark 
Where is `1.h`?

File: 1/8_11719_pjs.cpp [hunk 8]
```diff
@@ -0,0 +1,18 @@
+#include "1.h"
```
Comment (jukqaz) [inline 1/8_11719_pjs.cpp:1]: @korJiusngPark 
Where is `1.h`?

**Review by AlvinYou — APPROVED** [review]

**Review by cjy9306 — APPROVED** [review]
잉

**Review by jukqaz — APPROVED** [review]
When it commit from next time, add header.

File: 1/7_11718_pjs.cpp [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+	const auto strBegin = str.find_first_not_of(whitespace);
+	if (strBegin == std::string::npos)
+		return ""; // no content
+
+	const auto strEnd = str.find_last_not_of(whitespace);
+	const auto strRange = strEnd - strBegin + 1;
+
+	return str.substr(strBegin, strRange);
+}
+
```
Comment (kor-haru) [inline 1/7_11718_pjs.cpp:19]: This trim function is overrided ?

Comment (korJiusngPark) [reply 1/7_11718_pjs.cpp:19]: This trim function is maked by other people 
please, approve this PR

**Pull request merged**